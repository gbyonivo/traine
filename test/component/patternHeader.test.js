import React from 'react';
import { create } from 'react-test-renderer';
import PatternHeader from '../../src/components/patternHeader';

const props = {
  headerData: {
    destination: 'Better Naija',
    origin: 'Now Naija',
    operatedBy: 'Me and You'
  }
};

describe('patternHeader', () => {
  it('should render correctly', () => {
    const actual = create(<PatternHeader {...props}/>);
    expect(actual).toMatchSnapshot();
  });
});