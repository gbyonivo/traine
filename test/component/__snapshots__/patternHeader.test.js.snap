// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`patternHeader should render correctly 1`] = `
<div
  className="header"
>
  <div
    className="trainIcon"
  >
    <div
      className="trainIcon undefined undefined"
    />
  </div>
  <h4
    className="origin"
  >
    Now Naija
  </h4>
  <h3
    className="destination"
  >
    <span>
      to
    </span>
     
    Better Naija
  </h3>
  <div
    className="operatedBy"
  >
    Operated by 
    Me and You
  </div>
</div>
`;
